.github/workflows/re-build.yml

```diff
@@ -0,0 +1,34 @@
+name: Re-Build Non-appservice Images
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Re-release version (e.g. 4.9.0-rebuild.1)'
```

Comment:
update description  - re-publish

Reply:
I made it rebuild but I will change it to republish